knowledge_base/Environment-Variables.md

```diff
@@ -0,0 +1,394 @@
+
+## PGPASSWORD
+
+Postgres DB password (by default, `edgeware`). Bypasses usual password prompt for local DB commands.
+
+## PORT
+
+Localhost port location, default value `8080`.
+
+## PROCFILE
+
+Set in all of our Heroku apps that use a separate procfile. Specifies the path from the root of the repo to the Procfile to use for the deployed app. It is never needed locally but it must be set manually on Heroku and is required in production. In the production commonwealth app it is set to `packages/commonwealth/Procfile`.
+
+## RABBITMQ_API_URI
+
+This variable will default to the URI of a local RabbitMQ API server. This allows the use of the message publishing script to publish a message to a local RabbitMQ instance.
+
+## RABBITMQ_URI
+
+The URI of the RabbitMQ instance. On any Heroku app that has the CloudAMQP add-on (RabbitMQ provider), this is equal to CLOUDAMQP_URL. Locally, this variable defaults to the URI of a local RabbitMQ instance (Dockerized or native). Does not need to be set in a local environment unless you are spinning up the Discobot or Snapshot listener as well.
+
+## REDIS_URL
+
+Location to host a local Redis instance. By default `redis://localhost:6379`.
+
+## ROLLBAR_ENV
+
+A tag (string) used to identify Rollbar reports by environment, such that we can easily filter on the Rollbar dashboard. In staging and demo apps, we set it to the name of the app. Locally, it should be set to some custom value such as your name in lowercase. It should NEVER be set to `production`.
+
+## ROLLBAR_SERVER_TOKEN
+
+Refers to a specific project on Rollbar. Local and staging (i.e. non-prod) environments should always use the `CommonwealthDev` token, which may be obtained from the Rollbar dashboard, Timothee Legros or an engineering lead. Only prod uses the `Commonwealth` project token.
+
+## RPC_HOST
+
+Used in chain testing; for more information, see [Chain-Testing-Overview.md](../knowledge_base/Chain-Testing-Overview.md). By default, we use the value `ganache`.
+
+## SEND_WEBHOOKS_EMAILS
+
+Enables Webhook and email dispatching in production when set to `true`. Should be `false` or undefined elsewhere, to prevent Webhooks and emails from being sent in non-production environments. 
+
+## SENDGRID_API_KEY
+
+Used in email-based communications (notifications, digests, login).
+
+## SERVER_URL
+
+By default, this is set in our `commonwealth/server/config` file to `https://commonwealth.im` if in production, and `http://localhost:8080` otherwise.
+
+## SESSION_SECRET
+
+<!-- In need of documentation; if you have information about this token, please contribute. -->
+
+## SL_BUILD
+
+Boolean triggering our `snapshot-listener` package to build.
+
+## SL_PORT
+
+Port used for the `snapshot-listener` package. We use the default value `8001`.
+
+## SLACK_FEEDBACK_WEBHOOK
+
+<!-- In need of documentation; if you have information about this token, please contribute. -->
+
+Allows Slack users to send feedback via webhook.
```

Comment:
It actually enables users to share feedback on the app which shows up in our (old) slack via webhook.

Reply:
Do we monitor those channels or should we move to deprecate?

Reply:
We should monitor these channels, but we haven't updated the config to link it to the new slack... Been meaning to figure out how to do that.